blocks/library/latest-posts/index.js

```diff
@@ -35,6 +40,42 @@ registerBlockType( 'core/latestposts', {
 
 			this.latestPostsRequest
 				.then( latestPosts => this.setState( { latestPosts } ) );
+
+			this.toggleDisplayPostDate = this.toggleDisplayPostDate.bind( this );
+		}
+
+		toggleDisplayPostDate() {
+			const { displayPostDate } = this.props.attributes;
+			const { setAttributes } = this.props;
+
+			setAttributes( { displayPostDate: ! displayPostDate } );
+		}
+
+		componentWillReceiveProps( nextProps ) {
+			const { poststoshow: postToShowCurrent } = this.props.attributes;
+			let { poststoshow: postToShowNext } = nextProps.attributes;
+			const { setAttributes } = this.props;
+
+			postToShowNext = parseInt( postToShowNext );
```

Comment:
The `parseInt` is a bit weird, we should get the JSON attributes PR merged soon :)